src/components/policy/policy_external/src/policy_table/types.cc

```diff
@@ -704,22 +704,37 @@ void ModuleConfig::ReportErrors(rpc::ValidationReport* report__) const {
   if (!vehicle_year.is_valid()) {
     vehicle_year.ReportErrors(&report__->ReportSubobject("vehicle_year"));
   }
-  if (PT_PRELOADED == GetPolicyTableType()) {
-    std::string validation_info =
-        ommited_validation_info + PolicyTableTypeToString(GetPolicyTableType());
-    rpc::ValidationReport* ommited_field_report;
-    if (vehicle_make.is_initialized()) {
-      ommited_field_report = &report__->ReportSubobject("vehicle_make");
-      ommited_field_report->set_validation_info(validation_info);
-    }
-    if (vehicle_year.is_initialized()) {
-      ommited_field_report = &report__->ReportSubobject("vehicle_year");
-      ommited_field_report->set_validation_info(validation_info);
+
+  std::string validation_info =
+      ommited_validation_info + PolicyTableTypeToString(GetPolicyTableType());
+  rpc::ValidationReport* ommited_field_report;
```

Comment:
@Kozoriz I can't see why you need this value out of switch case scope, also why it is a pointer, you can use reference.